code/src/java/pcgen/gui2/tabs/models/HtmlSheetSupport.java

```diff
@@ -118,7 +105,7 @@ public void refresh()
 		}
 		if (refresher != null && !refresher.isDone())
 		{
-			refresher.cancel(false);
+			refresher.cancel(true);
```

Comment:
Is this an embedded bug fix, or ?

Reply:
yes. this is a bug fix. or at least a behavior change that I think is better. Will make it a separate PR for better future debugging.